src/Selection/RoundTranslation.js

```diff
@@ -0,0 +1,154 @@
+/* global gtag */
+import Modal_Selection from '../Modal/Selection.js';
+
+export default class Selection_RoundTranslation
+{
+    constructor(options)
+    {
+        this.baseLayout             = options.baseLayout;
+        this.markers                = options.markers;
+
+        this.step                   = Number.parseFloat(options.step);
+        this.offset                 = Number.parseFloat(options.offset);
+        this.roundX                 = Boolean(options.roundX);
+        this.roundY                 = Boolean(options.roundY);
+        this.roundZ                 = Boolean(options.roundZ);
+
+        this.selectionBoundaries    = (options.selectionBoundaries !== undefined) ? options.selectionBoundaries : null;
+        this.useHistory             = false;

```

Comment:
To enable this to be undo-able, would require a bit of work as all the objects' previous positions would need to be stored.